pkg/kubelet/network/dns/dns.go

```diff
@@ -207,7 +207,7 @@ func parseResolvConf(reader io.Reader) (nameservers []string, searches []string,
 	// Lines of the form "search example.com" overrule - last one wins.
 	searches = []string{}
 
-	// Lines of the form "option ndots:5 attempts:2" overrule - last one wins.
```

Comment:
This comment seems to assume overruling the last option was intentional. We should dig up the context for why we did this.

Reply:
Discussion in https://github.com/kubernetes/kubernetes/pull/54773 seems relevant.